setup-qubes.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# exit on errors, undefined variables, ensure errors in pipes are not hidden
+set -Eeuo pipefail
+
+PREFIX_APP_VM="A-"
+PREFIX_TEMPLATE_VM="Z-"
+
+# fetchFromVM VMNAME FILE [EXE]
+function fetchFromVm() {
+	if [ $# -lt 2 ]; then
+		echo "Expected at least two parameters: fetchFromVm SOURCEVMNAME FILENAME [EXE]"
+		return 1
+	fi
+	VMNAME="${1}"
+	FILE="${2}"
+	EXE="${3}"
+
+	echo "Fetching ${FILE} from VM ${VMNAME}..."
+	
+	# delete file in case it already exists on dom0 and ignore errors
+	FILENAME=$(basename "$FILE")
+	rm $FILENAME 2>>/dev/null
```

Comment:
Removing $FILENAME might fail, as the filepath for the file is not found when the $FILE contains subdirectories